sootup.callgraph/src/main/java/sootup/callgraph/RapidTypeAnalysisAlgorithm.java

```diff
@@ -250,17 +236,15 @@ protected void includeIgnoredCallsToClass(
     if (newEdges != null) {
       newEdges.forEach(
           call -> {
-            MethodSignature concreteTarget =
-                resolveConcreteDispatch(view, call.getTargetMethodSignature()).orElse(null);
-            if (concreteTarget == null) {
-              return;
-            }
-            addCallToCG(
-                call.getSourceMethodSignature(),
-                concreteTarget,
-                call.getInvokableStmt(),
-                cg,
-                workList);
+            resolveConcreteDispatch(view, call.getTargetMethodSignature())
+                .ifPresent(
+                    concreteTarget ->
+                        addCallToCG(
+                            call.getSourceMethodSignature(),
+                            concreteTarget,
+                            call.getInvokableStmt(),
```

Comment:
```suggestion
                            call.getInvokableStmt().asInvokableStmt(),
```
and then we could make the respective Parameter of addCallToCG more precise as well